.scalafmt.conf

```diff
@@ -0,0 +1,11 @@
+version=2.0.1
```

Comment:
I did not know that the version could be specified in the file, nice catch! 👍
This would allow also Idea (there is a plugin as you may know) to format with the same scalafmt version.

At this point it would make sense to remove the version from the Gradle build file, isn't it?

Reply:
Yes, I put this version here for the IDEA plugin. To my surprise, spotless does not read the version from the config. I replaced the spotless in the last commit with scalafmt-gradle and everything works as it should with the default settings